src/universalinit/templates/sqlite/config.yml

```
@@ -1,3 +1,7 @@
+configure_environment:
+  command: "python init_db.pyl"
```

I think we have a typo here
```suggestion
  command: "python init_db.py"
```